kie-wb-parent/kie-wb-runtime-webapp/src/main/java/org/kie/workbench/client/navigation/NavTreeDefinitions.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.workbench.client.navigation;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.dashbuilder.navigation.NavTree;
+import org.dashbuilder.navigation.impl.NavTreeBuilder;
+import org.kie.workbench.client.resources.i18n.NavigationConstants;
+
+import static org.dashbuilder.navigation.workbench.NavWorkbenchCtx.perspective;
+import static org.kie.workbench.common.workbench.client.PerspectiveIds.*;
+
+/**
+ * Navigation tree definitions such as the workbench menu bar
+ */
+@ApplicationScoped
+public class NavTreeDefinitions {
+
+    public static final String GROUP_ROOT = "root";
+    public static final String GROUP_WORKBENCH = "wb_group";
+
+    public static final String GROUP_DESIGN = "wb_group_design";
+    public static final String ENTRY_PROJECTS = "wb_entry_projects";
+    public static final String ENTRY_DASHBOARDS = "wb_entry_dashboards";
+
+    public static final String GROUP_DEVOPS = "wb_group_devops";
+    public static final String ENTRY_DEPLOYMENTS = "wb_entry_deployments";
+    public static final String ENTRY_EXECUTION_SERVERS = "wb_entry_execution_servers";
+
+    public static final String GROUP_MANAGE = "wb_group_manage";
+    public static final String ENTRY_PROCESS_DEFINITIONS = "wb_entry_process_definitions";
+    public static final String ENTRY_PROCESS_INSTANCES = "wb_entry_process_instances";
+    public static final String ENTRY_ADMINISTRATION_TASKS = "wb_entry_administration_tasks";
```

Comment:
We had a discussion the other day around proper naming of this perspective in https://github.com/kiegroup/kie-wb-distributions/pull/578.
I'm pasting here Jan's comment:
_"Aministration Tasks" sounds misleading to me, it sounds like some general application administration tasks, while in fact it's "just" admin view of task list -> IMHO Should be renamed to "Task Administration"_
This remark was accepted, so I think it would be good to be consistent with naming across all deliverables.